library/src/androidTest/java/net/danlew/android/joda/test/TestDateTimeZone.java

```diff
@@ -62,6 +62,7 @@
  */
 public class TestDateTimeZone extends InstrumentationTestCase {
     private static final boolean OLD_JDK;
+    private static final boolean JDK6;
```

Comment:
This fix should go into [the joda-time library](https://github.com/JodaOrg/joda-time). The tests here are just copied from there, so changing them here won't help much.